packages/contracts/contracts/fixtures/FixedResolution.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity 0.4.25;
+pragma experimental "ABIEncoderV2";
+
+import "../lib/Transfer.sol";
+
+
+contract FixedResolutionApp {
```

Comment:
To be absurdly explicit here, I would rename this to `ResolveToPay5ETHApp` or something like that. Alternatively, you could just reuse `PaymentApp` for your test case and store `[ addrA, 5, addr B, 0 ]` as the app state.

Reply:
I prefer not to use `PaymentApp` since there's some (unnecessary for a fixture IMO) indirection between the app instance and the app definition.

Reply:
Okay, I still recommend the extremely explicit name

Reply:
fixed